src/mockserver.js

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs';
+
+import { logger } from '@user-office-software/duo-logger';
+import { mockServerClient } from 'mockserver-client';
+
+async function mockserver() {
+    const respondToPostRequest = function (request) {
+        if (request.method !== 'POST') {
+            return;
+        }
+        let file;
```

Comment:
Now that we're only using `responsePath` in the `if` statements, there's no need to declare `file` here. It can be initialised at the end of the function as a `const`